src/App.jsx

```diff
@@ -13,14 +13,19 @@ const App = () => {
 
   // get the actor list initially on render
   // TODO: switch to useQuery to run simultaneously to the render instead of after initial render
-  useEffect(() => {
-    fetch('/actors')
-      .then((response) => response.json())
-      .then(setActors)
-      .catch((error) => {
-        console.error('Error: ', error);
-      });
-  }, []);
+  useEffect(() => { (async () => { // useEffect cannot take an async function. Must wrap async in regular function
```

Comment:
React throws an error when directly passing an async function to useEffect(). It suggests defining an async function within and calling it immediately as a work around. 
The syntax is a little strange, but essentially boilerplate and equivalent to this: 
```js
useEffect(function callback () {
    async function innerAsyncFunction() {
        // do stuff
    }

    innerAsyncFunction(); // call immediately
});
```